recipe/meta.yaml

```
@@ -115,6 +105,33 @@ outputs:
         - LICENSE
         - napari/_vendor/darkdetect/LICENSE
         - napari/_vendor/qt_json_builder/LICENSE
+      summary: a fast n-dimensional image viewer in Python, with only the required dependencies
+      doc_url: http://napari.org
+      dev_url: https://github.com/napari/napari
+
+  - name: napari
+    version: {{ version }}
+    build:
+      noarch: python
+    requirements:
+      run:
+        - {{ pin_subpackage('napari-base', exact=True) }}
+        - fsspec
+        - napari-plugin-manager >=0.1.0a2,<0.3.0a0
+        - napari-svg >=0.1.8
+        - numba >=0.57.1
+        - pooch >=1.3.0
+        # - py-triangle  # a bit fragile with bad data, leave out for now
```

```suggestion
        # - py-triangle  # a bit fragile with bad data, leave out for now
```

Context about fragility: https://napari.zulipchat.com/#narrow/stream/212875-general/topic/napari.20plugin.20manager.20in.20conda.20recipe/near/451712796

---

More context in napari/napari#6654

---

Oh, thanks, I'll add this as the comment instead